### PR TITLE
Signal/Level Crossing Animation Update

### PR DESCRIPTION
**Pull request by duncanspumpkin**
The remaining two animation updates

### CODE REVIEW

File: src/OpenLoco/src/Objects/LevelCrossingObject.h [hunk 1]
```diff
@@ -27,7 +27,8 @@ namespace OpenLoco
         uint8_t animationSpeed; // 0x07
         uint8_t closingFrames;  // 0x08
         uint8_t closedFrames;   // 0x09
-        uint8_t pad_0A[0x0C - 0x0A];
+        uint8_t var_0A;         // 0x0A
```
Comment (LeftofZen) [inline src/OpenLoco/src/Objects/LevelCrossingObject.h:30]: my favourite question - do you have any ideas for names/what these variables do? also `unk2` in the  enum below

**Review by LeftofZen — CHANGES_REQUESTED** [review]
The signals appear to function correctly but some signals do not visually update. The semaphore ones do update visually/animate, but the tricolour/modern ones (called "Colourlight" ingame) do not update visually at all.

File: src/OpenLoco/src/Objects/LevelCrossingObject.h (hunk 1 above)
Comment (LeftofZen) [reply src/OpenLoco/src/Objects/LevelCrossingObject.h:30]: also is there a var_0B or is that padding? or is 0A uint16?

File: src/OpenLoco/src/Objects/LevelCrossingObject.h [hunk 2]
```diff
@@ -27,7 +27,8 @@ namespace OpenLoco
...
+        uint8_t pad_0C;
```
Comment (LeftofZen) [inline src/OpenLoco/src/Objects/LevelCrossingObject.h:31]: this is 0x0B

**Review by LeftofZen — CHANGES_REQUESTED** [review]
variable is 0x0B, not  0C

File: src/OpenLoco/src/Objects/LevelCrossingObject.h (hunk 2 above)
Comment (duncanspumpkin) [reply src/OpenLoco/src/Objects/LevelCrossingObject.h:31]: Good catch

**Comment by duncanspumpkin** [comment]
> The signals appear to function correctly but some signals do not visually update. The semaphore ones do update visually/animate, but the tricolour/modern ones (called "Colourlight" ingame) do not update visually at all.

think i've fixed that now

**Review by LeftofZen — APPROVED** [review]

**Comment by LeftofZen** [comment]
> > The signals appear to function correctly but some signals do not visually update. The semaphore ones do update visually/animate, but the tricolour/modern ones (called "Colourlight" ingame) do not update visually at all.
> 
> think i've fixed that now

yep looks good to me

**Pull request merged**